extensions/codemagic/README.md

```diff
@@ -0,0 +1,5 @@
+# Codemagic
+See your Codemagic apps right in Raycast!
+
+# Setup
+Grab your token from [Codemagic settings](https://codemagic.io/settings) and you're good to go!
```

Comment:
It seems like it's better to tell users to do `Teams > Personal Account > Integrations > Codemagic API > Show` - the page you're linking is not the right place 🫤

Reply:
Oh, you're right! They must've changed it :/